apps/ejabberd/test/mongoose_rdbms_SUITE.erl

```diff
@@ -48,12 +51,30 @@ init_per_group(Group, Config) ->
 end_per_group(_, Config) ->
     Config.
 
+init_per_testcase(does_backoff_exceed, Config) ->
+    DbType = ?config(db_type, Config),
+    meck_config(DbType, ?KEEPALIVE_INTERVAL, ?MAX_INTERVAL),
+    meck_db(DbType),
+    meck_connection_error(DbType),
+    Config;
+init_per_testcase(does_backoff_increase, Config) ->
+    DbType = ?config(db_type, Config),
+    meck_config(DbType, 2, 10),
+    meck_db(DbType),
+    meck_connection_error(DbType),
+    Config;
 init_per_testcase(_, Config) ->
     DbType = ?config(db_type, Config),
-    meck_config(DbType, ?KEEPALIVE_INTERVAL),
+    meck_config(DbType, ?KEEPALIVE_INTERVAL, ?MAX_INTERVAL),
     meck_db(DbType),
     Config.
 
+end_per_testcase(T, Config) when T =:= does_backoff_icrease,
```

Comment:
> does_backoff_increase - why end_per_testcase does not unload everything?

Possibly because of typo in this line (does_backoff_i**n**crease)

Reply:
Thanks!

Reply:
No, that's not the case, @kzemek :(